js/console.go

```diff
@@ -47,9 +48,16 @@ func (c Console) log(ctx *context.Context, level log.Level, msgobj goja.Value, a
 
 	fields := make(log.Fields)
 	for i, arg := range args {
-		fields[strconv.Itoa(i)] = arg.String()
+		if arg.ExportType() == reflect.TypeOf((*[]byte)(nil)).Elem() {
```

Comment:
You can declare a global `var typeBytes = reflect.TypeOf(...)` to not have to recalculate this every time.